internal/trie/node/node.go

```diff
@@ -20,4 +22,5 @@ type Node interface {
 	SetGeneration(generation uint64)
 	Copy(copyChildren bool) Node
 	Type() Type
+	StringNode() (stringNode *gotree.Node)
```

Comment:
why not return the string directly? seem like both places StringNode are used it gets turned into a string?

Reply:
So it can print the entire trie nicely. You can also see the branch node uses `StringNode()` recursively on its children nodes.

It's really a pain to otherwise re-split lines and re-assemble, prefix with the right amount of indentation and right prefixes, that's why I wrote that tiny library over Christmas 😅

An alternative would be to have it as a function taking in a node interface and do type assertions? Or keep it as a method? I'm asking since this is only used for debugging really.

Reply:
ok, I missed that part, seems fine to me then